swagger/index.js

```diff
@@ -131,13 +133,21 @@ module.exports = yeoman.Base.extend({
           name: model.name,
           plural: model.plural,
           base: model.base || 'PersistedModel',
-          facetName: 'server', // hard-coded for now
+          facetName: 'common', // hard-coded for now
```

Comment:
Now the model definitions from Swagger are mapped to pure data models (sort of data transfer object) as the tagged operations are reflected into public models (controllers). It makes sense to use `common` instead of `server`.